### ISSUE TITLE
Linter config idea: Allow /* ... */ comments if they have certain text, like "Copyright"

### ISSUE THREAD

**Issue by srawlins** [brigade/scss-lint#427]
One legitimate use of `/* ... */` comments is preserving copyright notices.

How about a configurable whitelist, like:

```yaml
linters:
  Comment:
    allowed_multiline: Copyright
```

It could also be a list. And I'm not married to `allowed_multiline` as the config name. I'll send a PR if you like the idea.

**Comment by lencioni** [comment]
This is an interesting idea but it feels pretty awkward to configure and brittle. Why not just disable the linter for these special cases with the [inline comments](https://github.com/brigade/scss-lint#disabling-linters-via-source)?

```scss
// scss-lint:disable Comment
/* Copyright */
// scss-lint:enable Comment
```

**Comment by srawlins** [comment]
Thanks for the commenst, @lencioni.

Some companies might have a policy of placing a copyright notice at the top of every file, so the inline comment commands would not scale.

What do you think would be brittle about this idea? I can see the list idea maybe being brittle, and definitely awkward. I considered allowing regexen but that's even more extreme: just way too awkward.

I think the example I have above is very concise, obvious as to its intention, and not brittle, at least with the example word "Copyright." I can see it maybe being brittle with... different words there? Maybe... I dunno, let me know what you think.

**Comment by sds** [comment]
I think a regex-based solution would be fine here. The `Comment` linter can add a `allowed` option which accepts a regex, so for this case you can have:

```yaml
linters:
  Comment:
    allowed: '^\s*Copyright'
```

This allows arbitrary comments to be ignore, satisfying whatever policy a team requires.

A pull request with tests would be welcome!

**Comment by srawlins** [comment]
Hrm, so the regex idea presents one problem in that the CommentNode that Sass provides us has the entire syntax of the comment as the value of the node. So:

```scss
/*
 * Copyright someone
 */
a { color: red; }
```

contains a CommentNode with a value of `"/*\n * Copyright someone\n */"`, which would not match `'^\s*Copyright`. And I don't think there is a clean way to strip the comment "text" from it's syntax. For example, if I strip all asterisks at the beginning of a line, then I would strip these asterisks, unintentionally:

```scss
/*
Comment paragraph, and a list:

* item 1
* item 2
*/
```

So maybe we should not encourage use of anchors (`^`, `\A`, etc.).

**Comment by sds** [comment]
Whatever regex suits your needs. I provided it merely as an example, but thanks for clarifying that it doesn't work in this case.

**Comment by sds** [comment]
Implemented by #431. Thanks!
